_source/user-guide/infrastructure-monitoring/managing-metrics-account.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: article
+title: Managing your Metrics account
+permalink: /user-guide/infrastructure-monitoring/metrics-account.html
+flags:
+  admin: true
+  logzio-plan:
+tags:
+contributors:
+  - shalper
+---
+
+By default, your Metrics account is found under your main log monitoring account.
+To open it, log into your Logz.io Operations account and select the
```

Comment:
1. `To open it` - not clear if `it` refers  to your logging or your metrics account
2. No such thing as "Operations account" - change to `Operations workspace`